### PR TITLE
Remove notebooks UI

### PR DESCRIPTION
**Pull request by sindhuba**
This change removes the notebooks component UI in Portal Frontend as part of the feature deprecation.
[Preview this branch](https://cosmos-explorer-preview.azurewebsites.net/pull/1779?feature.someFeatureFlagYouMightNeed=true)

### CODE REVIEW

File: src/Explorer/Menus/CommandBar/CommandBarComponentButtonFactory.test.ts [hunk 1]
```diff
@@ -368,62 +366,6 @@ describe("CommandBarComponentButtonFactory tests", () => {
     });
   });
 
-  describe("GitHub buttons", () => {
```
Comment (sevoku) [inline src/Explorer/Menus/CommandBar/CommandBarComponentButtonFactory.test.ts:371]: Why are we commenting out the other changes but fully removing this one?

**Review by sevoku — CHANGES_REQUESTED** [review]
It'd be good to add written comments to each change, about why we comment out the code (as simple as "//deprecated notebooks UI"). And also either comment out the code as whole or remove it, mixing is confusing when reviewing code afterwards (without the need to read git history)

File: src/Explorer/Menus/CommandBar/CommandBarComponentButtonFactory.test.ts (hunk 1 above)
Comment (sindhuba) [reply src/Explorer/Menus/CommandBar/CommandBarComponentButtonFactory.test.ts:371]: I got rid of the tests but in the code - I will ultimately check to see if there are any dependencies/references to the commented portion so I can remove them easily all together. So, it'd be better for me to track it this way.

**Comment by sindhuba** [comment]
> It'd be good to add written comments to each change, about why we comment out the code (as simple as "//deprecated notebooks UI"). And also either comment out the code as whole or remove it, mixing is confusing when reviewing code afterwards (without the need to read git history)

Got it - removed all the code to avoid confusion in the future. Thanks.

**Review by jawelton74 — APPROVED** [review]

**Review by sevoku — COMMENTED** [review]
Hhhm, thnking twice... shouldn't we remove the actual notebooks code as well? Isn't it all dead code now sitting in the codebase?

**Pull request merged**